src/CodeGenerator/src/Generator/ResponseParser/RestJsonParser.php

```diff
@@ -112,15 +112,15 @@ private function getInputAccessorName(Member $member)
         return $member->getLocationName();
     }
 
-    private function parseElement(string $input, Shape $shape, bool $required)
+    private function parseElement(string $input, Shape $shape, bool $required, bool $inObject = false)
```

Comment:
Let's make this non optional. Also, can we add a PHP doc block on this method to describe `$inObject`?

Reply:
fixed